{{cookiecutter.project_slug}}/clients/mobile/react-native/src/stores/navigation.ts

```diff
@@ -0,0 +1,4 @@
+import type { MyNavio } from '@screens/routes'
+import { atom } from 'jotai'
+
```

Comment:
Where does this get set? It seems to me it is always null

Reply:
True I think I missed that line in the navio file